packages/react-core/src/demos/ComposableMenu/examples/ComposableMultipleTypeaheadSelect.tsx

```diff
@@ -0,0 +1,219 @@
+    if (inputValue) {
+      newMenuItems = intitalMenuItems.filter(menuItem =>
+        String(menuItem.children)
+          .toLowerCase()
+          .includes(inputValue.toLowerCase())
+      );
+
+      // When no options are found after filtering, display 'No results found'
+      if (!newMenuItems.length) {
+        newMenuItems = [{ isDisabled: true, children: 'No results found' }];
+      }
+    }
+
+    setMenuItems(newMenuItems);
+  }, [inputValue]);
+
+  const handleMenuArrowKeys = (key: string) => {
+    let indexToFocus;
+
+    if (isMenuOpen) {
+      if (key === 'ArrowUp') {
+        // When no index is set or at the first index, focus to the last, otherwise decrement focus index
+        if (focusedItemIndex === null || focusedItemIndex === 0) {
+          indexToFocus = menuItems.length - 1;
+        } else {
+          indexToFocus = focusedItemIndex - 1;
+        }
+      }
+
+      if (key === 'ArrowDown') {
+        // When no index is set or at the last index, focus to the first, otherwise increment focus index
+        if (focusedItemIndex === null || focusedItemIndex === menuItems.length - 1) {
+          indexToFocus = 0;
+        } else {
+          indexToFocus = focusedItemIndex + 1;
+        }
+      }
+
+      setFocusedItemIndex(indexToFocus);
+    }
+  };
+
+  const onInputKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    const enabledMenuItems = menuItems.filter(menuItem => !menuItem.isDisabled);
+    const [firstMenuItem] = enabledMenuItems;
+    const focusedItem = focusedItemIndex ? enabledMenuItems[focusedItemIndex] : firstMenuItem;
+
+    switch (event.key) {
+      // Select the first available option
+      case 'Enter':
+        if (!isMenuOpen) {
+          setIsMenuOpen(prevIsOpen => !prevIsOpen);
+        } else {
+          onMenuSelect(focusedItem.itemId as string);
+        }
+        break;
+      case 'Tab':
+      case 'Escape':
+        setIsMenuOpen(false);
+        break;
+      case 'ArrowUp':
+      case 'ArrowDown':
+        handleMenuArrowKeys(event.key);
+        break;
+      default:
+        !isMenuOpen && setIsMenuOpen(true);
+    }
+  };
+
+  // Close the menu when a click occurs outside of the menu toggle content
+  const onDocumentClick = (event: MouseEvent) => {
+    if (
+      isMenuOpen &&
+      !menuToggleRef.current?.contains(event?.target as HTMLElement) &&
+      !menuRef?.current?.contains(event.target as Node)
+    ) {
+      setIsMenuOpen(false);
+    }
+  };
+
+  const toggleMenuOpen = () => {
+    setIsMenuOpen(!isMenuOpen);
+    textInputRef.current?.focus();
+  };
+
+  const onTextInputChange = (value: string) => {
+    setInputValue(value);
+  };
+
+  const onMenuSelect = (itemId: string) => {
+    if (itemId) {
+      setSelected(
+        selected.includes(itemId) ? selected.filter(selection => selection !== itemId) : [...selected, itemId]
+      );
+    }
+  };
+
+  const toggle = (
+    <MenuToggle
+      variant="typeahead"
```

Comment:
This doesn't appear to be working - maybe you just need to rebase? Seems to work fine on main.

The typeahead looks like this with extra padding around the text input
<img width="923" alt="Screen Shot 2022-09-01 at 8 25 02 PM" src="https://user-images.githubusercontent.com/35148959/188039686-9eea6fdf-20e6-44a7-9622-33919c069022.png">

It should look like this - 36px tall (matches all the other form stuff and buttons)

<img width="867" alt="Screen Shot 2022-09-01 at 8 25 15 PM" src="https://user-images.githubusercontent.com/35148959/188039683-0e82719c-7834-40e4-bed8-48a8e49e8262.png">
<img width="805" alt="Screen Shot 2022-09-01 at 8 28 00 PM" src="https://user-images.githubusercontent.com/35148959/188039685-239a4b77-6f31-460a-997f-832bb63fb508.png">

Reply:
@mcoker yes a rebase should fix that.  Jeff fixed the styling issue in this PR #7905